### DISCUSSION TITLE
Non-hidden config file

### DISCUSSION THREAD

**Discussion by sourcejedi** [ansible/ansible-lint#1284] [Ideas]
ansible-galaxy lets you override their lint rules, by creating an `.ansible-lint` file in your role. Yay!

This effectively means part of your role's tests are stored in a "hidden" unix file. Boo!

For now, I've symlinked `.ansible-lint` to `ansible-lint.cfg`. This means I can see the config file.

Could ansible-lint start reading `ansible-lint.cfg` ?

**Comment by ssbarnea** [comment]
I am considering switching the configuration file to `.config/ansible-lint.yaml` as this should also follow the XDG standards and resolve the issue of cluttering the project root config giles.

By switching I mean: deprecating, and adding a warning which asks user to rename the file.

Due to the re nature of this change, I will wait for extra feedback before doing anything.
